src/composables/tippingContracts.ts

```diff
@@ -0,0 +1,75 @@
+import { computed, ref } from '@vue/composition-api';
+import { Contract } from '@aeternity/aepp-sdk-13';
+
+import TippingV1ACI from '../lib/contracts/TippingV1ACI.json';
+import TippingV2ACI from '../lib/contracts/TippingV2ACI.json';
+import { RUNNING_IN_TESTS } from '../lib/environment';
+import { useSdk13 } from './sdk13';
+import { watchUntilTruthy } from '../popup/utils';
+import {
+  IDefaultComposableOptions,
+  INetwork,
+  TippingV1ContractApi,
+  TippingV2ContractApi,
+} from '../types';
+
+interface TippingContracts {
+  tippingV1: Contract<TippingV1ContractApi>;
+  tippingV2?: Contract<TippingV2ContractApi>;
+}
+
+let tippingV1: Contract<TippingV1ContractApi> | undefined;
+let tippingV2: Contract<TippingV2ContractApi> | undefined;
+const initializing = ref(false);
```

Comment:
This also doesn't have to be reactive.

Reply:
I see that you followed my comment, but I think I was wrong, because there is a `watchUntilTruthy` that is using this value and it requires it to be reactive.